crates/typst-cli/src/args.rs

```diff
@@ -494,6 +494,7 @@ display_possible_values!(DiagnosticFormat);
 #[derive(Debug, Copy, Clone, Eq, PartialEq, ValueEnum)]
 pub enum Feature {
     Html,
+    PdfAccessibility,
```

Comment:
Not sure how this will serialize with `ValueEnum`, but kebab-case would be desirable.

I am also not sure about the flag name because it seems to imply that accessibility as a whole is guarded.

How about `a11y-extras`?